geom/geom/inc/TGeoManager.h

```diff
@@ -115,6 +115,8 @@ private :
    typedef NavigatorsMap_t::iterator                         NavigatorsMapIt_t;
    typedef std::map<std::thread::id, Int_t>                  ThreadsMap_t;
    typedef ThreadsMap_t::const_iterator                      ThreadsMapIt_t;
+   // Map of constant properties
+   typedef std::map<TString, Double_t>                       ConstPropMap_t;
```

Comment:
`map<string, double>` has already a dictionary in ROOT: can that type can be used?

Reply:
Yes, thanks, done